src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,77 @@
+package duke.tasks;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.File;
+
+
+public class Task {
+
+    private final String name;
+    private boolean done;
```

Comment:
Perhaps naming it isDone would be better, since it helps to understand later that the variable is boolean.